GetRepoJSON/getRepoJSON.py

```diff
@@ -0,0 +1,52 @@
+# repos.py
+
+# A demo script that demonstrates api calls /user
+
+
+
+from time import sleep
+def delayserver():
+    sleep(0.01)
+    return
+
+
+import sys
+sys.path.append('..')
```

Comment:
😧 I understand why this is here, but this causes me anguish.  Probably not worth dealing with it for these scripts.  Python modules are stupidly complicated.

Reply:
I actually had a question about the best way to do this. Some of the functionality I can move to py-allspice, but I wasn't sure the best way to share common functions in a python project. I'm happy to rewrite if you know of a better way.

Reply:
When you run a script with `python foo/bar.py`, the directory `foo` gets added to the path.  On the other hand, if you run the script with `python -m foo.bar` from the repo root, the current working directory, the repo root in this case, gets added to the path.  It is also aware of its package, so relative imports can work.  Even though both ways run the same Python script, only the latter leads to intuitive imports (both absolute and relative) that work consistently, whether installed with pip or not, without mucking with `sys.path`.

A way to work around this is to only ever run scripts at the top level.  The top-level scripts can import and call whatever nested module they want.  Then the two are equivalent.  This is what we do in AllSpice-Tools.  But I understand that this can be unsatisfactory.  One could argue that the habit of running things with `python foo/bar.py` should be stopped in favor of `python -m foo.bar`.  However, that would also mean that you can't run the file directly as in `./foo/bar.py` with a shebang, and shell completion no longer works.  That's a bummer.

Reply:
I removed this reference. The new py-allspice module removes the reason to use this.